src/components/CallView/shared/VideoBottomBar.vue

```diff
@@ -212,8 +212,8 @@ export default {
 		},
 		audioButtonTooltip() {
 			return this.model.attributes.audioAvailable
-				? t('spreed', 'Mute')
```

Comment:
OK, so please check my last commit.

Reply:
Again, it was correct (as per screenshots).
`this.model.attributes.audioAvailable === true` => button is active => tooltip indicates action 'Mute'
`this.model.attributes.audioAvailable === false` => button is disabled => tooltip indicates state 'Muted'

Am I missing something?

Reply:
I'm sorry, I must be pretty tired already at the beginning of the week. It's not your fault, it's mine ; my neurons aren't doing their job properly 😅, because I don't quite understand our conversation. 

Just tell me what I need to change and I'll make a final commit.

Reply:
I don't think there's something needed to be changed)

If microphone is ON, tooltip should be "Mute".
If microphone is OFF, tooltip should be "Muted".

Reply:
Ok. But thanks for the effort anyway! If you'll find something else, don't hesitate to drop us a message in community chats